.rubocop.yml

```diff
@@ -3,11 +3,58 @@ AllCops:
   Exclude:
     - 'bundler/**/*'
     - 'lib/rubygems/resolver/molinillo/**/*'
+    # TODO: Remove all Exclude values after this line.
```

Comment:
Why?

Reply:
Oh! When I first worked on this, I was getting multiple thousands of RuboCop warnings from code under `test/`. Apparently other changes I made drove that down to 20.

I'll remove this line + the one below it when I continue work on this PR, and make a follow-up PR resolving those.